app/views/layouts/application.html.erb

```diff
@@ -2,6 +2,8 @@
 <html>
   <head>
     <title>MaintenanceApp</title>
+    <link rel="stylesheet"
```

Comment:
this should be fine, but normally try to use rails helpers when you can. example here: https://stackoverflow.com/questions/18508996/include-google-font-in-rails-app